internal/tsetse/plugin.ts

```diff
@@ -0,0 +1,41 @@
+import * as tsForTypes from 'typescript';
+import * as tsServerLibForTypes from 'typescript/lib/tsserverlibrary';
+
+import * as perfTrace from '../tsc_wrapped/perf_trace';
```

Comment:
I don't think perfTrace is useful in this context because we don't run the nodejs process, so we wouldn't know where to find the resulting file?

Reply:
Done, just blindly copy pasting :-)